src/site/content/en/blog/a11y-tips-for-web-dev/index.md

```diff
@@ -0,0 +1,557 @@
+
+<div class="w-table-wrapper">
+  <table>
+    <thead>
+      <tr>
+        <th>Vision</th>
+        <th>Hearing</th>
+        <th>Mobility</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>
+          <ul>
+            <li>Low vision</li>
+            <li>Blindness</li>
+            <li>Color blindness</li>
+            <li>Cataracts</li>
+            <li>Sun glare</li>
+          </ul>
+        </td>
+        <td>
+          <ul>
+            <li>Hard of hearing</li>
+            <li>Deafness</li>
+            <li>Noise</li>
+            <li>Ear infection</li>
+          </ul>
+        </td>
+        <td>
+          <ul>
+            <li>Spinal cord injury</li>
+            <li>Limited dexterity</li>
+            <li>Hands full</li>
+          </ul>
+        </td>
+      </tr>
+      <tr>
+        <th>Cognitive</th>
+        <th>Speech</th>
+        <th>Neural</th>
+      </tr>
+      <tr>
+        <td>
+          <ul>
+            <li>Learning disability</li>
+            <li>Sleepiness or distraction</li>
+            <li>Migraine</li>
+            <li>Autism</li>
+            <li>Seizure</li>
+          </ul>
+        </td>
+        <td>
+          <ul>
+            <li>Ambient noise</li>
+            <li>Sore throat</li>
+            <li>Speech impediment</li>
+            <li>Unable to speak</li>
+          </ul>
+        </td>
+        <td>
+          <ul>
+            <li>Depression</li>
+            <li>PTSD</li>
+            <li>Bipolar disorder</li>
+            <li>Anxiety</li>
+          </ul>
+        </td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+**Visual issues** range from an inability to distinguish colors to no vision at all.
+
+- Ensure text content meets a minimum
+  [contrast ratio threshold](http://www.w3.org/TR/WCAG20/#visual-audio-contrast-contrast).
+- Avoid communicating information
+  [using solely color](http://www.w3.org/TR/2008/REC-WCAG20-20081211/#visual-audio-contrast-without-color)
+  and ensure that all text is
+  [resizable](http://www.w3.org/TR/2008/REC-WCAG20-20081211/#visual-audio-contrast-scale).
+- Ensure all user interface components can be used with assistive technologies
+  such as screen readers, magnifiers, and braille displays.
+  This entails ensuring that UI components are marked up
+  such that accessibility APIs can programmatically determine
+  the _role_, _state_, _value_, and _title_ of any element.
+
+**Tip:** The element inspection feature in the Chrome, Edge, and Firefox developer
+tools displays a tooltip of CSS properties
+that includes a quick check for color contrast ratio.
+
+![Screenshot of the Chrome DevTools Inspect Element tooltip.](./inspect-element.jpg)
+
+I personally live with low vision and am embarrassed to say
+I'm that person who always zooms in on sites, their DevTools, and terminal.
+While supporting zoom is almost never at the top of anyone's list,
+optimizing for low-vision users is always appreciated… 🤓
```

Comment:
@addyosmani, I'm wondering whether we want to imply that living with a disability is something to be embarrassed about? What if we rephrased this paragraph to something like: "I personally live with low vision, and I often find myself zooming in on sites, their DevTools, and the terminal. While supporting zoom is almost never at the top of developers' to-do lists, it can make a world of difference to users like me."

That way, you're still getting the personal appeal while side-stepping the negative connotation. WDYT?

Reply:
Made this edit.